galaxyui/src/app/utilities/score/score.component.ts

```diff
@@ -1,9 +1,12 @@
 import { Component, Input, OnInit } from '@angular/core';
 
-class RepoScore {
+interface IRepoScore {
     community_score: any;
-    quality_score: any;
+    quality_score?: any;
```

Comment:
Why is the quality score now optional?

Reply:
This component was originally supposed to accept a `Repository` type which has `community_score`, `quality_score` and `community_survey_count` attributes. Collections also need to be able to use this component now and collections don't a `quality_score` attribute.

Modifying it so that it either accepts a `quality_score` or `latest_version` object was the easiest way to make it so it accepts collections and repositories.